repos/spack_repo/builtin/packages/plasma/package.py

```diff
@@ -42,13 +42,15 @@ class Plasma(CMakePackage):
         "17.1",
         sha256="d4b89f7c3d240a69dfe986284a14471eec4830b9e352ae902ea8861f15573dee",
         url="https://github.com/icl-utk-edu/plasma/releases/download/17.01/plasma-17.01.tar.gz",
+        deprecated=True,
     )
 
     build_system(
         conditional("makefile", when="@:17.1"),
         conditional("cmake", when="@18.9:"),
         default="cmake",
     )
+    depends_on("cmake@:3", when="build_system=cmake", type="build")
```

Comment:
The first argument should be `cmake@3.12.0:` according to PLASMA's `CMakeLists.txt`